allennlp/modules/token_embedders/pretrained_transformer_embedder.py

```diff
@@ -101,6 +101,13 @@ def __init__(
             model_name,
             tokenizer_kwargs=tokenizer_kwargs,
         )
+
+        try:
+            self.transformer_model.resize_token_embeddings(len(tokenizer.tokenizer))
```

Comment:
I'm assuming this is a no-op if the size already matches?

Reply:
Yes, nothing would happen if no additional tokens are added.